gensim/models/hdpmodel.py

```diff
@@ -445,11 +444,17 @@ def show_topic(self, topic_id, num_words=20, log=False, formatted=False):
         `False` as lists of (weight, word) pairs.
 
         """
+        if topn is None: #deprecated num_words is used
+            logger.warn("The parameter num_words for show_topic() method would be deprecated in the updated version.\
+            Please use topn instead. Ignore if you didn't use parameter num_words or topn for show_topic() ")
```

Comment:
what is the purpose of adding "ignore if"? Would it be better to make `topn=20` by default and `num_words=None`. Show warning `if num_words is not None` and add a comment to make it an Exception in the next release. Same applies everywhere.